blocks/lead-art-block/features/leadart/leadart.scss

```diff
@@ -53,4 +52,5 @@
 		vertical-align: middle;
 		width: 100%;
 	}
+	@include block-margin-bottom;
```

Comment:
I don't think this is appropriate.  This adds margin tot he bottom of the block and we should be using the gaps in layout.

Reply:
did we intend on updating this block with this story?

Reply:
got this stylelint issues https://github.com/WPMedia/arc-themes-blocks/actions/runs/3387865884